packages/core/src/components/AvatarGroup/__stories__/AvatarGroup.stories.tsx

```diff
@@ -383,14 +396,15 @@ export const GridTooltip = {
   )
 };
 
-export const CounterCustomTooltipContent = {
+export const CounterCustomTooltipContent: Story = {
   render: () => (
     <AvatarGroup
       size={Avatar.sizes.LARGE}
       type={Avatar.types.IMG}
       max={4}
       counterTooltipCustomProps={{
-        content: "... and plenty more employees"
+        content: "... and plenty more employees",
+        forceRenderWithoutChildren: false
```

Comment:
Is it needed? I think a better solution would be to use `Partial<TooltipProps>` for `counterTooltipCustomProps` in AvatarGroup.tsx (and AvatarGroupContainer.tsx)